zipkin_thrift_span_test.go

```diff
@@ -263,8 +264,10 @@ func TestThriftLocalComponentSpan(t *testing.T) {
 		thriftSpan := BuildZipkinThrift(sp)
 
 		anno := findBinaryAnnotation(thriftSpan, "lc")
-		assert.NotNil(t, anno)
+		require.NotNil(t, anno)
 		assert.EqualValues(t, test.wantAnnotation, anno.Value)
+		require.NotNil(t, anno.Host)
+		assert.NotEqual(t, 0, anno.Host.Ipv4)
```

Comment:
if `Ipv4` is not of type `int`, then this test always succeeds. I try to avoid using NotEqual for this reason, safer to do a `==` check and `t.Error()`